pkg/cloudprovider/aws/amifamily/custom.go

```diff
@@ -59,3 +59,7 @@ func (c Custom) ENILimitedMemoryOverhead() bool {
 func (c Custom) PodsPerCoreEnabled() bool {
 	return true
 }
+
+func (c Custom) EvictionSoftEnabled() bool {
```

Comment:
I think all these switches for enabling kubelet pass throughs could be put into Options (in resolver.go) so that everything defaults to `true` except for the ones we need to override like BR.

Reply:
I re-labeled it as `FeatureFlags`

Reply:
ahh fancy! love it